### ISSUE TITLE
There should be a one-click option to copy an existing Shorty URL

### ISSUE THREAD

**Issue by arkascha** [owncloud/shorty#88]
The only purpose of creating a Shorty is to use that Shorty URL to post it or otherwise share it. 
For this it makes sense to be able to access (copy) that URL as simple and fast as possible: one single click. 

Problem here: cross browser copy-to-clipboard does not (yet) work.

**Comment by fredl99** [comment]
I have no problems with copy&paste when using the "share link" button within the table. (Linux / any browser)

What I'd rather like to see would be an extension to easily create a shorty for any file in the "Files" section of OC. But maybe that would justify a separate feature request.

**Comment by arkascha** [comment]
Certainly copying the link from within the share button works. But things can be sped up: 
click a line (an Shorty), maybe navigate to the desired entry (keyboard shortcuts) and press CTRL-C. 

----
Creating a shorty for an OC-internal object is a separate feature request, indeed. 
This particular request has been discussed many times before. Unfortunately this is much more complicated than it appears at first glance. Reason behind this is that the OC-Sharing-API simply does not offer such feature.

**Comment by fredl99** [comment]
I couldn't find any discussion in this matter.
Maybe you could borrow some code from the "files move" app? I know I'm sounding kind of naive ;)

**Comment by arkascha** [comment]
Sorry, could I ask you to open a separate issue for that separate request? Thanks!
Some tracks of the discussion can be seen in the comments to the main Shorty app inside the app store. In addition I remember some email communication, but I am not sure if that was on a mailing list or direct. Sorry.

**Comment by fredl99** [comment]
You're right, sorry for my ignorance. I've opened a new one and deleted my according postings here.

**Comment by arkascha** [comment]
:green-heart:
